kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java

```diff
@@ -210,6 +210,44 @@ public String getProcessInstanceImage(String containerId, Long processInstanceId
         }
     }
 
+    @Override
+    public String getProcessInstanceImageCustomColor(String containerId, Long processInstanceId, String completeNodeColor,
+                                                     String completeNodeBorderColor, String activeNodeColor) {
+        if (config.isRest()) {
+            Map<String, Object> valuesMap = new HashMap<String, Object>();
+            valuesMap.put(RestURI.CONTAINER_ID, containerId);
+            valuesMap.put(RestURI.PROCESS_INST_ID, processInstanceId);
+
+            StringBuffer params = new StringBuffer();
+            params.append(RestURI.SVG_NODE_COMPLETED_COLOR + "=").append(encode(completeNodeColor));
+            params.append("&" + RestURI.SVG_NODE_COMPLETED_BORDER_COLOR + "=").append(encode(completeNodeBorderColor));
+            params.append("&" + RestURI.SVG_NODE_ACTIVE_COLOR + "=").append(encode(activeNodeColor));
+
+            Map<String, String> headers = new HashMap<String, String>();
+            headers.put("Accept", MediaType.APPLICATION_SVG_XML);
+
+            return makeHttpGetRequestAndCreateRawResponse(
+                    build(loadBalancer.getUrl(), IMAGE_URI + "/" + PROCESS_INST_IMG_GET_URI + "?" + params.toString(),
+                          valuesMap), headers);
+        } else {
+            CommandScript script = new CommandScript(Collections.singletonList(
+                    (KieServerCommand) new DescriptorCommand("ImageService",
+                                                             "getActiveProcessImage",
+                                                             new Object[]{containerId,
```

Comment:
```suggestion
                                                             new Object[]{
                                                                 containerId,
```

Reply:
@cristianonicolai not sure if I applied well the suggestion, I just put the first value at the next line but it doesn't fit just under Object word like I understand you are proposing. I can do it manually but I looks bad fight against the IDE code formatting.